docs(snap): Update camera interface, getting started doc version (#143)
edgexfoundry-holding/device-usb-camera: snap/README.md

Please refer [here][secret-store-token] for further information.

### Connect the camera interface
The [`camera`](https://snapcraft.io/docs/camera-interface) interface is currently **NOT automatically connected** as it is pending auto-connection permissions from the store.

To connect manually:
```
snap connect edgex-device-usb-camera:camera :camera
```

> 

Reviewer: This isn't obsolete. The connection should still happen manually when the snap is built from source.

Please refer [here][secret-store-token] for further information.

### Connect the camera interface

> The [`camera`](https://snapcraft.io/docs/camera-interface) interface provides automatic access to all cameras.

Reviewer: ```suggestion
The [`camera`](https://snapcraft.io/docs/camera-interface) interface grants access to USB cameras.
When installing the snap from the store, the access is granted automatically. However, if the snap is built and installed locally, the interface will not auto-connect. You can check the connection status by running the following command: `snap connections edgex-device-usb-camera`.
```

> However, if the snap is built and installed locally, the interface will not auto-connect. You can check the connection status by running the following command: `snap connections edgex-device-usb-camera`.

Reviewer: ```suggestion
```

To manually connect:
```
snap connect edgex-device-usb-camera:camera :camera
```

> Please refer [here][device-usb-camera] for further information.

Reviewer: ```suggestion
Please refer to [docs] for further information.
```

### Example of use

> Please refer [here][edgex-demo] for an example of setting up the  edgex-device-usb-camera snap, including its configuration and commands for starting the stream.

Reviewer: The documentation has coverage for these.

[edgex-device-usb-camera]: https://snapcraft.io/edgex-device-usb-camera
[docs]: https://docs.edgexfoundry.org/3.0/getting-started/Ch-GettingStartedSnapUsers/#device-usb-camera
[secret-store-token]: https://docs.edgexfoundry.org/3.0/getting-started/Ch-GettingStartedSnapUsers/#secret-store-token

> [device-usb-camera]: https://docs.edgexfoundry.org/3.0/getting-started/Ch-GettingStartedSnapUsers/#device-usb-camera

Reviewer: Redundant reference.
```suggestion
```

> [edgex-demo]: https://github.com/canonical/edgex-demos/tree/main/openvino-object-detection#2-edgex-setup-device-usb-camera

Reviewer: ```suggestion
```
